dnf-behave-tests/dnf/downgrade.feature

```diff
@@ -184,104 +182,51 @@ Scenario: Downgrade mixture of not available/not installed/not downgradable/down
     And stderr is
     """
     No match for argument: nosuchpkg
-    Package "wget.x86_64" of lowest version already installed, cannot downgrade it.
+    The lowest available version of the "wget.x86_64" package is already installed, cannot downgrade it
```

Comment:
There is a missing dot at the end. The test doesn't pass.
 It is also missing in two other tests.